community/examples/hpc-slurm-local-ssd-v5-legacy.yaml

```
@@ -37,6 +37,14 @@ deployment_groups:
     settings:
       local_mount: /home
 
+  - id: startup
+    source: ./modules/scripts/startup-script
+    settings:
+      local_ssd_filesystem:
+        fs_type: "ext4"
+        mountpoint: "/mnt/localssd"
+        permissions: "0755"
```

I think it's worth being explicit to users about the string format for the filesystem permissions. Without the quotes, it gets parsed to the number 493. Removing the quotes on strings is a `nit`, but I prefer the quoteless style where possible as it makes YAML more human legible than machine-readable JSON.

```suggestion
        fs_type: ext4
        mountpoint: /mnt/localssd
        permissions: "0755" # must quote numeric filesystem permissions!
```

---

Done